### PR TITLE
Distinct orderby

### PR DESCRIPTION
**Pull request by lightyeare**
Write up post on Distinct OrderBy problem we ran in to the other day.

Also includes my author stuff

### CODE REVIEW

File: _posts/2016-11-11-distinct-orderby-gotcha.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+date: 2016-11-11 09:53:43
+tags: 
+- Linq
+- .NET
+categories: Development
+twitter_text: "OrderBy Distinct Gotcha"
+authors: Chad Peters
+image: https://c2.staticflickr.com/9/8146/7127935057_2d9bf9c1b8_k.jpg
+image_url: https://www.flickr.com/photos/khawkins04/
+image_credit: Ken HawkIns
```
Comment (kendaleiv) [inline _posts/2016-11-11-distinct-orderby-gotcha.md]: Could you lowercase the `I` in the last name here?

File: _posts/2016-11-11-distinct-orderby-gotcha.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                PlanId = plan.PlanId,
+                Benefit = benefit.Benefit,
+                SentencesSortOrder = benefit.SentencesSortOrder,
+             })
+             .Distinct() //Distinct must preceed OrderBy
+             .OrderBy(x => x.SentencesSortOrder);
+```
+
+## Conclusion
+Be careful when using Distinct() and OrderBy() on IQueryables. If you don't specify them in the correct order you may not get the results you expect.
```
Comment (kendaleiv) [inline _posts/2016-11-11-distinct-orderby-gotcha.md]: Want to wrap `Distinct()` and `Order()` in backticks?

File: _posts/2016-11-11-distinct-orderby-gotcha.md (hunk 1 above)
Comment (lightyeare) [inline _posts/2016-11-11-distinct-orderby-gotcha.md]: Nice font, Flickr. i thought it was a capital i.

**Comment by khalidabuhakmeh** [comment]
Damn the format got busted because there weren't new lines

**Pull request merged**